bears/r/FormatRBear.py

```diff
@@ -15,8 +20,44 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         r_max_expression_length: int=20):
+        """
+        :param r_keep_comments:         Use to determine whether comments are
+                                        kept or not.
+        :param r_keep_blank_lines:      Use to determine whether blank lines
+                                        are kept or not.
+        :param r_braces_on_next_line:   Use to determine wheter a brace should
+                                        be placed on the next line.
+        :param r_use_arrows:            Use to determine if either the assign
+                                        operator ``=`` or the arrow ``<-``
+                                        should be used.
+        :param tab_width:               Number of space for indentation.
+        :param r_max_expression_length: Maximum number of characters for an
+                                        expression. Note (different from the
+                                        maximum number of characters for
+                                        a line).
```

Comment:
What note? :D
And I think you mean "names" instead of "expressions", as an expression can be formed out of many variables and stuff. Then you don't need to note that this is different frmo max-chars-per-line^^